requirements.txt

```diff
@@ -0,0 +1,5 @@
+opentracing>=2.0
```

Comment:
Both `django-opentracing` and `wavefront-opentracing-sdk-python` 
have this dependency, do we need this one here, explicitly?